gattc_linux.go

```diff
@@ -222,7 +222,21 @@ func (s DeviceService) DiscoverCharacteristics(uuids []UUID) ([]DeviceCharacteri
 // writes can be in flight at any given time. This call is also known as a
 // "write command" (as opposed to a write request).
 func (c DeviceCharacteristic) WriteWithoutResponse(p []byte) (n int, err error) {
-	err = c.characteristic.Call("org.bluez.GattCharacteristic1.WriteValue", 0, p, map[string]dbus.Variant(nil)).Err
+	args := make(map[string]any)
+	err = c.characteristic.Call("org.bluez.GattCharacteristic1.WriteValue", 0, p, args).Err
+	args["type"] = "command"
+	if err != nil {
+		return 0, err
+	}
+	return len(p), nil
+}
```

Comment:
```suggestion
	args := map[string]any{"type": "command"}
	if err := c.characteristic.Call("org.bluez.GattCharacteristic1.WriteValue", 0, p, args).Err; err != nil {
		return 0, err
	}

	return len(p), nil
}
```
I simply have no experience with it, so this is only a suggestion for better code style and technical correctness - with my assumptions after reading this document: https://kernel.googlesource.com/pub/scm/bluetooth/bluez/+/5.46/doc/gatt-api.txt

Of course, "n" and "err" in line 224 should be removed afterwards.